ncf

```diff
@@ -51,11 +52,34 @@ def init(args):
 
   print("INFO: Success")
 
+def write_everything(path, force):
+  write_all_methods(path)
+  write_all_techniques(path,force)
+
+def write_all_methods(path):
+  methods = ncf.get_all_generic_methods_metadata()
+  with open('/var/rudder/configuration-repository/ncf/generic_methods.json', 'w') as outfile:
+    json.dump(methods, outfile, indent=2)
+
+def write_all_techniques(path, force):
+  techniques = ncf.get_all_techniques_metadata(True,force)
+  for bundle_name in techniques["data"]["techniques"]:
+    technique = techniques["data"]["techniques"][bundle_name]
+    category = technique["category"] if len(technique["category"]) > 0 else "ncf_techniques"
+    version = technique["version"] if len(technique["version"]) > 0 else "1.0"
+    techniqueFile = '/var/rudder/configuration-repository/techniques/'+category+"/"+ bundle_name +"/"+version+'/technique.json'
+    if not os.path.exists(techniqueFile):
```

Comment:
what will update this file when it exists?

Reply:
If it's already written then it will be managed by Rudder (in fact the goal here is to bootstrap the technique.json file so that Rudder will enrich them)